assets/scripts/app/__tests__/PrintContainer.test.js

```diff
@@ -20,14 +21,14 @@ describe('PrintContainer', () => {
     const { asFragment } = renderWithRedux(<PrintContainer />, {
       initialState: { app: { printing: false } }
     })
-    expect(asFragment.querySelector('.print-container')).toBeInTheDocument()
     expect(asFragment()).toMatchSnapshot()
```

Comment:
I took out the above line since it didn't seem to match the react-testing library principles, and a snapshot test would be pretty small here (basically an empty print container) and match the other 'it renders' tests we have

Reply:
Ok, that's fine